tools/CLU/BuildPackage.ps1

```diff
@@ -35,9 +35,9 @@ if ($packageId -ne "Microsoft.CLU.Commands")
 {
     $contentFileText += "    <file src=""content\*.lx"" target=""content""/>`r`n"
     $contentFileText += "    <file src=""content\*xml"" target=""content""/>`r`n"
-    if (Test-Path "$packageSource\content\help") 
+    if (Test-Path "$packageSource\content\content\help") 
```

Comment:
When I try to build it ... I don't see those help files getting copied  to drop/... . I found out it's creating the help files in src\bin\...\publish\..\content\content\help. It was more like a fix than a intentional change. Can you please try the previous version of this script and see if that was able to copy the help files properly?